developer_docs.md

```diff
@@ -98,6 +100,16 @@ pip install --upgrade pip setuptools
 pip install -r requirements.txt
 ```
 
+### Run schema tests on local machine
+
+If your system has not already done so [activate a python environment](#create-and-activate-a-python-virtual-environment), and then [install requirements](#install-requirements).
+
```

Comment:
Rather than repeating the instructions for setting up a local development environment in each how-to guide, I prefer to reference them once at the beginning of the section on proposing changes and to make sure the relevant how-to guides are mentioned there. So I would add a 'Run tests' step to those instructions, between running `./manage.py pre-commit` and building the documentation.

```suggestion
```

Reply:
This is actually a bigger issue I have with the dev docs as they currently stand. It's not clear in the "Propose changes" top section what the exact steps are for running the pre-commit, e.g. that you need to activate the python environment and install the requirements, and I think these steps also need to be clear when explaining how to run the tests.  I'm going to make some changes to both incorporate your point here and also hopefully address the one I've just made.

Reply:
Ah, that makes sense. I've made some further updates to reduce repetition.

Reply:
I also reordered the how-to guides to reflect the order in propose changes.